src/functions/TestResults.NUnit25.ps1

```diff
@@ -110,6 +110,7 @@ function Write-NUnitTestSuiteElements {
 
     $suites = @(
         # Tests only have GroupId if parameterized. All other tests are put in group with '' value
+        # PowerShell 6.1+ sorts by default in Group-Object. We need to sort for consistent output in Windows PowerShell
```

Comment:
This says we sort but the code below does not use sort? Is this a mistake?

Reply:
Very likely. Good catch :)